### ISSUE TITLE
[4.8.0] Tree support ignores areas requiring support

### ISSUE THREAD

**Issue by war4peace** [Ultimaker/Cura#9389]
**Application version**
Cura 4.8.0, Arachne Engine Beta

**Platform**
Windows 10 X64, 32 GB DDR4, CPU Intel i7 6800K, 2x 1080 Ti in SLI

**Printer**
Creality CR 10 MAX with modifications (Direct Drive, etc)

**Reproduction steps**
1. Bought a Yoda model from CGTrader (https://www.cgtrader.com/3d-models/miniatures/figurines/yoda-sculpture-3d-print-model)
2. Resized to 200mm Z height (locked axes)
3. Sliced using Tree Supports option, his left hand (the one holding the cane), palm area is never supported, as if the algorithm completely ignores it. Also, Yoda left ear is ignored when 

**Screenshot(s)**
Model showing problematic area (4.8.0):
![image](https://user-images.githubusercontent.com/6294884/110947970-785acd00-8349-11eb-92db-bb378dac7289.png)

Left hand claws shown being printed in mid-air (4.8.0):
![image](https://user-images.githubusercontent.com/6294884/110948510-19498800-834a-11eb-9415-78be147d9bc7.png)

First layer of left ear, shows as being printed horizontally (parallel to ground) but no support shown (4.8.0)
![image](https://user-images.githubusercontent.com/6294884/110948717-5ca3f680-834a-11eb-9b15-3ccba6266d07.png)

Model after slicing, showing lack of supports (4.8.0)
![image](https://user-images.githubusercontent.com/6294884/110947708-2e71e700-8349-11eb-898d-cabef8f3de36.png)

Same issues in Arachne Beta:
![image](https://user-images.githubusercontent.com/6294884/110948233-c4a60d00-8349-11eb-93d6-a87c5bbb5c5f.png)

**Actual results**
No supports in area clearly requiring supports

**Expected results**
Supports in that area.

**Project file**
This is a paid model, I don't think it would be wise to share it publicly (breach of license).
I can, however, transfer the project to a representative privately, for debugging purposes.

**Log file**
Not applicable

**Additional information**
I have tried "Support Overhang Angle" values starting at 65 and dropping all the way to 40, with the same result. Tree support algorithm ignores areas where support is absolutely required.

LATER EDIT:
If I change the "Support Overhang Angle" to 45 degrees, the left ear is now supported, but the right ear is no longer supported. Also, although I have clearly selected supports "Touching buildplate", there are tree supports being generated in mid-air, above the model's right shoulder.

![image](https://user-images.githubusercontent.com/6294884/110953607-3d0fcc80-8350-11eb-856a-f3565e1d66ad.png)

**Comment by fvrmr** [comment]
Hi @war4peace thank you for your report.
I just sliced a familiar model (found on Thingiverse) and it wasn't a problem for me. 

![Screenshot (109)](https://user-images.githubusercontent.com/77000924/110964259-4c444980-8353-11eb-8c54-9ad6e670049f.png)

So I am really curious to your project file. I saw that you've posted this also on a Facebook group. I will contact you there.

**Comment by war4peace** [comment]
@fvrmr Hi, There are some differences between how we were slicing. I have not chosen "Everywhere" under "Support Placement" because the model doesn't seem to require it, and the last thing I want is for supports to affect the model's upper surfaces.
At any rate, I have sent you a link to the model on Facebook Messenger.

**Comment by fvrmr** [comment]
It looks like this has to do with the Z distance and this is already a known bug, see here: https://github.com/Ultimaker/Cura/issues/8566

The work around I found for this model is changing the the `support x/y distance` to 0.4 mm the `support overhang angle` to 45 degrees and the `tree support collision resolution` to 0.4mm. Then I added a support blocker to the places that don't need support. 

An other solution is add custom support to the places there is no support created. 

I will close this issue since it is a duplicate. But feel free to reach out for more questions.
